app/assets/javascripts/fixedsticky.js

```diff
@@ -0,0 +1,191 @@
+;(function( win, $ ) {
```

Comment:
is the js frm a lib? if it is , let's put it in vendor http://guides.rubyonrails.org/asset_pipeline.html#asset-organization